BUILD/task_order/Quantum Terrarium.dat

```diff
@@ -1,20 +1,11 @@
 LX_freeCombatsTask
 catBurglarHeist
 auto_breakfastCounterVisit
-chateauPainting
-LX_faxing
-LX_artistQuest
-LX_galaktikSubQuest
-LX_armorySideQuest
-LX_meatsmithSubQuest
```

Comment:
galaktik might be automatically triggered if for some reason MC is very poor, like if they have no IOTMs and missing a lot of perms. and have enabled the setting to let it automatically trigger.
so I do not think it should be removed.

he other 3 basically user setting configured and by default are not done. No harm of leaving them there though.